rosdep/base.yaml

```diff
@@ -1400,13 +1401,15 @@ libdrm-dev:
       packages: [x11-base/nouveau-drm]
   ubuntu: [libdrm-dev]
 libesd0-dev:
+  debian: [libesd0-dev]
   fedora: [esound-devel]
   ubuntu: [libesd0-dev]
 libespeak-dev:
   debian: [libespeak-dev]
   fedora: [espeak-devel]
   ubuntu: [libespeak-dev]
 libestools-dev:
+  debian: [libestools2.1-dev]
```

Comment:
rosdep doesn't support pure virtual packages, can you use the parent package  `libestools-dev` (https://packages.debian.org/stretch/libestools-dev) instead ?